accelerator/config.c

```diff
@@ -111,6 +111,13 @@ status_t cli_core_set(ta_core_t* const core, int key, char* const value) {
         ta_log_error("Malformed input or illegal input character\n");
       }
       break;
+    case CACHE_ACQUIRE:
+      ta_log_info("Initializing cache state\n");
```

Comment:
It is not initializing here, we initialize the cache state in `ta_core_default_init`.

Reply:
If the caching service was not enabled, then we don't need to initialize the cache object.

Reply:
I don't understand the relation between my comment and your reply.

Reply:
`ta_core_default_init()` initialize only the arguments of cache object, but it dosen't initialize caching service.

Reply:
We initialize the cache state to `false` in `ta_core_default_init()` but not here.

Reply:
We have set state of cache as `false` in it